jmx/src/test/java/org/jboss/as/jmx/model/MBeanInfoFactoryTestCase.java

```diff
@@ -0,0 +1,2 @@
+package org.jboss.as.jmx.model;public class MbeanInfoFactoryTestCase {
```

Comment:
@kabir This class breaks compilation. The file name uses MBean (capital B) while the classname uses Mbean (lower case b).

It looks like it's cruft regardless.